site/src/atomics/flex.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Flex Atomics
+description: Flex related atomic css classes for the Atlas Design System
+template: standard
+---
+
+# Flex Atomics
+
+Flex atomics are useful when composing layouts and patterns containing other components.
+
+| cssproperty       | value                                                                               | screensize |
+| ----------------- | ----------------------------------------------------------------------------------- | ---------- |
+| `flex-direction`  | `row`, `column`, `row-reverse`, `column-reverse`                                    | `tablet`   |
+| `justify-content` | `flex-start`, `flex-end`, `center`, `space-around`, `space-between`, `space-evenly` | `tablet`   |
+| `align-items`     | `flex-start`, `flex-end`, `center`, `baseline`, `stretch`                           | `tablet`   |
+| `align-self`      | `flex-start`, `flex-end`, `center`, `baseline`, `stretch`                           | `tablet`   |
+| `flex-wrap`       | `nowrap`, `wrap`, `wrap-reverse`                                                    | `tablet`   |
+| `align-content`   | `flex-start`, `flex-end`, `space-around`, `space-between`, `stretch`                | `tablet`   |
+| `flex-grow`       | `0`, `1`                                                                            | `tablet`   |
+| `flex-shrink`     | `0`, `1`                                                                            | `tablet`   |
+
+## Usage
+
+The following example uses a combination of mobile-first flex atomics to achieve a responsive card layout.
+
+```html
+<article
+	id="example-layout"
+	class="background-color-body-medium border border-radius-lg display-flex flex-direction-row"
+>
+	<div
+		class="background-color-info color-info-invert padding-md-tablet border-radius-lg display-flex align-items-center flex-shrink-0 flex-grow-0 "
```

Comment:
Add `padding-xxs` in addition to `padding-md-tablet` so there is still a little space around on mobile

Reply:
The h2 sit on top now so can just stick with `padding-md`.